app/controllers/results_controller.rb

```diff
@@ -4,6 +4,8 @@ class ResultsController < ApplicationController
   REGION_WORLD = "world"
   YEARS_ALL = "all years"
   SHOW_100_PERSONS = "100 persons"
+  SHOWS = ['mixed', 'slim', 'separate', 'history', 'mixed history'].freeze
```

Comment:
I think RuboCop doesn't like this...?

Reply:
I was also expecting it to correct it, but I think because `mixed history` has a space it's fine with not using %w? It clearly didn't complain in CI, at any rate :P